apps/web/app/(use-page-wrapper)/settings/(settings-layout)/organizations/(org-admin-only)/attributes/sync/page.tsx

```diff
@@ -0,0 +1,63 @@
+import { _generateMetadata } from "app/_utils";
+
+import IntegrationAttributeSyncView from "@calcom/features/ee/integration-attribute-sync/IntegrationAttributeSyncView";
+import { getIntegrationAttributeSyncService } from "@calcom/features/ee/integration-attribute-sync/di/IntegrationAttributeSyncService.container";
+import { TeamRepository } from "@calcom/features/ee/teams/repositories/TeamRepository";
+import SettingsHeader from "@calcom/features/settings/appDir/SettingsHeader";
+import { PrismaAttributeRepository } from "@calcom/lib/server/repository/PrismaAttributeRepository";
+import { prisma } from "@calcom/prisma";
+
+import { validateUserHasOrgPerms } from "../../../actions/validateUserHasOrgPerms";
+
+export const generateMetadata = async () =>
+  await _generateMetadata(
+    (t) => t("attribute_sync"),
+    (t) => t("attribute_sync_description"),
+    undefined,
+    undefined,
+    "/settings/organizations/attributes/sync"
+  );
+
+const Page = async () => {
+  // PBAC permission check - redirect if user doesn't have permission
+  const session = await validateUserHasOrgPerms({
+    permission: "organization.attributes.create",
```

Comment:
Thought is if a user has permission to adjust attributes then they'll also have permissions to setup this attribute sync